### DISCUSSION TITLE
Displaying text alongside the preload progress bar

### DISCUSSION THREAD

**Discussion by arvidguterstam** [jspsych/jsPsych#759] [Q&A] [answered]
Hi everyone,

In my experiment I'm preloading 150MB of video files using the through jsPsych.init(). Is there any way to display a text message alongside the preload progress bar (eg "Preloading might takes a couple of mins depending on your connection speed, so please be patient...)?

I've received feedback from subjects that some of them thinks that webpage has crashed because of the long preloading duration, so a message assuring them that this is not the case would probably be helpful.

Thanks for a fantastic product!

**Comment by jodeleeuw** [comment, marked as answer]
This is a great idea for a feature. We don't currently have a simple parameter to add that would allow this. But you can modify `jspsych.js` to do it.

The relevant lines are here:

https://github.com/jspsych/jsPsych/blob/83980085ef604c815f0d97ab55c816219e969b84/jspsych.js#L2548-L2553

Adding a message to this HTML should be safe. I don't expect it to cause any conflicts with other bits of code.

**Reply by arvidguterstam** [reply]
Thank you for the extremely rapid answer. This solved my issue!

**Reply by wangzhuoranivy** [reply]
Hi. I'm wondering how the message can be added in between. I added the text in this line like
`pb_html += "<div id='jspsych-loading-progress-bar' style='height: 10px; width: 0%; background-color: #777;'>Images are loading...</div>"; ` 
but it doesn't show up...

Thanks!
